source/views/covid/updates-row.js

```diff
@@ -23,7 +23,11 @@ export class UpdatesRow extends React.PureComponent<Props> {
 
 	render() {
 		const {update} = this.props
-		const posted = moment(update.datePublished).format('MMM Do YYYY')
+
+		const posted = moment
+			.utc(update.datePublished)
+			.utcOffset(-6)
+			.format('MMM Do YYYY')
```

Comment:
Would love some feedback on this.

Reply:
To be clear, `MMM Do YYYY` produces date strings like

- `Oct 3rd 2020`
- `Jan 12th 2007`
- etc.

Is there enough room to do `October 3rd, 2020`?  For display to humans, I think that's the most readable/fancy-feeling, but if there's a width constraint I'd go with more compact representations like `10/03/2020` (cringe) or `2020-10-03` (which confuses people apparently).  I also like `3rd Oct, 2020` (just a transposition) but it seems kinda weird in certain contexts. Or, if we want to get _really_ fancy, `moment` seems to have a [`fromNow`](https://momentjs.com/docs/#/displaying/fromnow/), so we could get strings like `6 days ago`, which are fairly compact but vague.

Reply:
There's no width constraint so I'm open to changing it. The website currently follows this format: `1 October 2020` but I like your suggestion with the shorter month?

Reply:
Of the four, do we like one of these the best?

~ | ~
--|--
![long](https://user-images.githubusercontent.com/5240843/95111780-dcbb9580-06f4-11eb-8159-619d9c4a3321.png) | ![transposed](https://user-images.githubusercontent.com/5240843/95111784-de855900-06f4-11eb-9db6-262cc3b5a9d3.png)
![short](https://user-images.githubusercontent.com/5240843/95111790-e04f1c80-06f4-11eb-8648-481a5c2b3a40.png) | ![image](https://user-images.githubusercontent.com/5240843/95111840-ef35cf00-06f4-11eb-92c5-9766603d3309.png)

Reply:
MMM DD, YYYY (3 days ago)?

Reply:
Going with `MMMM D, YYYY` (3 days ago)`!